src/dialog/an-dialog.factory.js

```diff
@@ -42,8 +42,21 @@
 
                 var onFocusIn = function (ev) {
                     var modalElement = $body.find('.an-dialog-wrapper')[0];
+                    var target = ev.target;
 
-                    if (stack.length && !modalElement.contains(ev.target)) {
+                    if (!stack.length) {
+                        return;
+                    }
+
+                    // In case the target element belongs to select2, dont't do anything.
+                    // Select2 injects the options list into the html body, so it is outside of the modal,
+                    // even if the select2 itself is used inside the modal.
+                    if (angular.element.fn && angular.element.fn.closest &&
```

Comment:
I only added the check so that the code doesn't break in case jQuery is missing. Angulars build-in jqLite does not seem to support closest. But I agree that it is a quite unlikely case.